soapui/src/main/resources/com/eviware/soapui/explorer/soapui-pro-api-endpoint-explorer-starter-page.html

```diff
@@ -166,6 +165,16 @@
         });
 
         var initPage = function () {
+            window.propertyUpdater = {
```

Comment:
@AlexanderNikitin-Smartbear if we going to make universal solution for updating values in webview, this script should be injected to the page from Java.

Reply:
Do we really need a universal solution in SoapUI OS? If it takes too much time, let's postpone final polishing until we move this solution into ReadyAPI.

Reply:
@AlexanderNikitin-Smartbear I'm agree with @antone-sb, we can postpone it.

Reply:
I see the most difficult task to guaranty that injection from Java and calling this injection from page are synchronized.